types/identicon.js/index.d.ts

```diff
@@ -6,6 +6,8 @@ export interface IdenticonOptions {
     margin?: number | undefined;
     size?: number | undefined;
     format?: "svg" | "png" | undefined;
+    saturation?: number | undefined;
+    brightness?: number | undefined;
 }
 
 export interface PNGlib {
```

Comment:
regarding the `export default` error, since identicon.js uses CommonJS modules, `@types/identicon.js` needs to use them too. It looks like tests are passing now without that change, but if you want to do it, you need to

1. change `export default class Identicon { ...` to `class Identicon { ...` then `export = Identicon`.
2. Wrap all the other interfaces and types in `namespace Identicon { ...`
3. In tests, `import Identicon = require('identicon.js')` and refer to `Svg` as `Identicon.Svg`.

Reply:
Thanks for the answer, I'll probably submit another pr to address this when I have time